src/index.js

```diff
@@ -0,0 +1,63 @@
+import itemsCount from './modules/itemsCount.js';
+import postLike, { getLikes } from './modules/likes.js';
+import './style.css';
+
+// eslint-disable-next-line no-unused-vars
+const homeContainer = document.querySelector('.homepage');
+const itemsCountSpan = document.querySelector('.recipe-count');
+const baseUrl = 'https://www.themealdb.com/api/json/v1/1/search.php?f=b';
+
+const displayMeals = (meals) => {
+  const borderDiv = document.createElement('div');
+  borderDiv.className = 'mainCont';
+  // borderDiv.style.backgroundColor = 'red';
+  // eslint-disable-next-line array-callback-return
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.
- [ ] please don't disable linters we use them to make our code more efficient and easier to read.